src/data/data2d/Data2DManager.js

```diff
@@ -24,4 +24,24 @@ export class Data2DManager {
     });
     return ob;
   };
+
+  static fromBruker = function fromBruker(result, options = {}) {
+    let data = result.minMax;
+
+    const datum2D = new Datum2D({
+      ...options,
+      meta: result.info,
+      data,
+      source: {
+        jcamp: null,
+        jcampURL:
+          options.source && options.source.jcampURL
+            ? options.source.jcampURL
```

Comment:
How there can be a jcampURL if it is coming from Bruker ? I think this field should be null.

Reply:
It is done already